### PR TITLE
5d support for interpolate onnx_linear mode

### PR DESCRIPTION
**Pull request by chenhu-wang**
ticket: 43100

### CODE REVIEW

**Comment by chenhu-wang** [comment]
@dmitry-gorokhov could you please have a review?

**Review by dmitry-gorokhov — APPROVED** [review]
CPU part looks good to me

File: ngraph/core/src/op/interpolate.cpp [hunk 1]
```diff
@@ -185,7 +185,7 @@ void op::v4::Interpolate::infer_using_scales(PartialShape& output_shape,
         if (padded_input_shape[axis].is_static())
         {
             float padded_len = static_cast<float>(padded_input_shape[axis].get_length());
-            int64_t new_dim = static_cast<int64_t>(padded_len * scales[i] + epsilon);
+            int64_t new_dim = static_cast<int64_t>(padded_len * (scales[i] + epsilon));
```
Comment (dmitry-gorokhov) [inline ngraph/core/src/op/interpolate.cpp:188]: @vgavrilo Could you please approve changes in ngraph ref?

Comment (vgavrilo) [reply ngraph/core/src/op/interpolate.cpp:188]: It is a correct change.

**Pull request merged**